mail/user_invitation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use app\modules\v1\users\models\Invitation;
+use yii\helpers\Html;
+use yii\helpers\Url;
+
+/** @var Invitation $model */
+?>
+
+<p><?php echo Yii::t('app', 'Hello {name}', ['name' => $model->email]) ?></p>
+<p>
+    <?php echo Yii::t('app', 'You were invited into {name} by {inviterName}', [
+        'name' => Yii::$app->name,
+        'inviterName' => $model->createdBy->getDisplayName()
+    ]) ?>
+</p>
+<p>
+    <?php echo Yii::t('app', 'Click the link bellow to register') ?>
+	<br>
+    <?php echo Html::a(Html::encode(Yii::t('app', 'Registration Link')),
+        Url::to('@frontend/auth/register/' . $model->token)) ?>
```

Comment:
@skavt Please use array instead of direct string + do not use `Url::to` here case `Html::a` calls `Url::to` inside.